transport/src/main/java/io/netty/channel/DefaultChannelId.java

```diff
@@ -53,7 +53,10 @@
 
     private static final AtomicInteger nextSequence = new AtomicInteger();
 
-    static ChannelId newInstance() {
+    /**
+     * Create and return a new {@link DefaultChannelId} instance.
```

Comment:
I still think `Returns a new ...` implies that it creates a new instance and returns it. i.e.

- 'create' is redundant.
- It should be 'returns' instead of 'return', because it's not an imperative sentence.